dev.Dockerfile

```diff
@@ -0,0 +1,18 @@
+# ARG BASE_IMAGE
+# ARG BASE_IMAGE_VERSION
+
+# debian image
+# FROM python:3.8-slim-bullseye
+FROM ghcr.io/bento-platform/bento_base_image:python-debian-latest
+
+RUN mkdir /beacon
+
+COPY ./entrypoint.dev.sh /beacon/bento_beacon/entrypoint.dev.sh
+COPY ./requirements.txt /beacon/bento_beacon/requirements.txt
+
+# shared volume with local repo, see docker.compose.dev.yaml
+WORKDIR /beacon/bento_beacon
+
+RUN pip install debugpy -r requirements.txt; 
+
```

Comment:
it looks like the project files are never added to the dev dockerfile

Reply:
currently the repo itself has some default config files that would be copied over here, although it's probably better to have dev and prod handle config in the same way.

Reply:
config should come from environment variables as much as possible... maybe changing the config python file to use environment vars for almost everything with normal development defaults and using `python_dotenv` to load env vars from a gitignored `.env` file for development for any needed overrides would be a good way to go

Reply:
In this case "config" includes large json files for detailed information about beacon behaviour that can change for each instance, such as details of which endpoints are served and what kind of behaviour you can expect. That's why I opted to copy files at build time. There are a couple of these that could be paramaterized a bit better so that passing an entire file is unnecessary, but not sure yet if that will work for all of them.

Reply:
Ah, in that case the path to the JSON can be passed via env var maybe and then docker volumes should take care of it in production...

Reply:
Thats the purpose of `CONFIG_ABSOLUTE_PATH`